--- mecab-0.996/src/dictionary.orig.cpp	2023-01-29 19:54:21
+++ mecab-0.996/src/dictionary.cpp	2023-01-29 19:54:44
@@ -179,14 +179,6 @@
 
   cid.open(left_id_file.c_str(),
            right_id_file.c_str(), &config_iconv);
-  CHECK_DIE(cid.left_size()  == matrix.left_size() &&
-            cid.right_size() == matrix.right_size())
-      << "Context ID files("
-      << left_id_file
-      << " or "
-      << right_id_file << " may be broken: "
-      << cid.left_size() << " " << matrix.left_size() << " "
-      << cid.right_size() << " " << matrix.right_size();
 
   std::ofstream ofs(output);
   CHECK_DIE(ofs) << "permission denied: " << output;
